assets/js/show-task.js

```diff
@@ -23,6 +24,10 @@ function showTask(inputValue, formattedDate, taskList, form, allTasks, id) {
   const date = document.createElement("p");
   date.innerHTML = "Created At: " + formattedDate;
 
+  const saveButton = document.createElement("button");
+  saveButton.innerText = "Save";
+  saveButton.classList.add(HIDE_CLASS);
```

Comment:
`createButton` [factory](https://github.com/rsa-wtag/Todo-App-with-JS/blob/d66c018aec158a1080164237993b5623dac1a553/assets/js/factory/createButton.js#L1) should be reused in order to avoid code duplication. 

Mandatory read (Manifesto): [Don’t Repeat Yourself](https://docs.google.com/document/d/1iEgKrcrt0_QNiz5Ywmy2_K55xXDC9B1YkyJWx17Py_s/edit)